### ISSUE TITLE
Different results when querying with diferent language preferences

### ISSUE THREAD

**Issue by hallpower** [komoot/photon#514]
I've submitted following request via the browser: https://photon.komoot.io/api/?q=muscat&osm_tag=aeroway:aerodrome

This yields following result: 

```json
{
    "features": [
        {
            "geometry": {
                "coordinates": [
                    58.29247709617641,
                    23.5987194
                ],
                "type": "Point"
            },
            "properties": {
                "city": "Al Mawalih ash Shamaliyah",
                "country": "Oman",
                "countrycode": "OM",
                "extent": [
                    58.2477853,
                    23.6155156,
                    58.329042,
                    23.5833579
                ],
                "name": "Muscat International Airport",
                "osm_id": 20440333,
                "osm_key": "aeroway",
                "osm_type": "W",
                "osm_value": "aerodrome",
                "postcode": "1122",
                "state": "Muscat",
                "street": "Way 808",
                "type": "house"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -91.14591628475205,
                    41.3660709
                ],
                "type": "Point"
            },
            "properties": {
                "city": "Muscatine",
                "country": "United States of America",
                "countrycode": "US",
                "county": "Muscatine County",
                "extent": [
                    -91.1606258,
                    41.3753627,
                    -91.1283817,
                    41.3590896
                ],
                "name": "Muscatine Municipal Airport",
                "osm_id": 104449438,
                "osm_key": "aeroway",
                "osm_type": "W",
                "osm_value": "aerodrome",
                "postcode": "52749",
                "state": "Iowa",
                "type": "house"
            },
            "type": "Feature"
        },
        {
            "geometry": {
                "coordinates": [
                    -91.2198191,
                    41.586144
                ],
                "type": "Point"
            },
            "properties": {
                "country": "United States of America",
                "countrycode": "US",
                "county": "Muscatine County",
                "name": "Walker Field",
                "osm_id": 1042011245,
                "osm_key": "aeroway",
                "osm_type": "N",
                "osm_value": "aerodrome",
                "postcode": "52776",
                "state": "Iowa",
                "street": "Hayes Avenue",
                "type": "house"
            },
            "type": "Feature"
        }
    ],
    "type": "FeatureCollection"
}
```

When I run the same request via Postman (or cURL), I get a different result set: 
```bash
curl --location --request GET 'https://photon.komoot.io/api/?q=muscat&osm_tag=aeroway:aerodrome'
```

```json
{
    "features": [
        {
            "geometry": {
                "coordinates": [
                    -91.14591628475205,
                    41.3660709
                ],
                "type": "Point"
            },
            "type": "Feature",
            "properties": {
                "osm_id": 104449438,
                "extent": [
                    -91.1606258,
                    41.3753627,
                    -91.1283817,
                    41.3590896
                ],
                "country": "United States of America",
                "city": "Muscatine",
                "countrycode": "US",
                "postcode": "52749",
                "county": "Muscatine County",
                "type": "house",
                "osm_type": "W",
                "osm_key": "aeroway",
                "osm_value": "aerodrome",
                "name": "Muscatine Municipal Airport",
                "state": "IA"
            }
        },
        {
            "geometry": {
                "coordinates": [
                    -91.2198191,
                    41.586144
                ],
                "type": "Point"
            },
            "type": "Feature",
            "properties": {
                "osm_id": 1042011245,
                "osm_type": "N",
                "country": "United States of America",
                "osm_key": "aeroway",
                "street": "Hayes Avenue",
                "countrycode": "US",
                "osm_value": "aerodrome",
                "postcode": "52776",
                "name": "Walker Field",
                "county": "Muscatine County",
                "state": "Iowa",
                "type": "house"
            }
        }
    ],
    "type": "FeatureCollection"
}
```

I've tried changing the user-agent, but to no avail. What am I missing? Thanks for your support.

**Comment by simonpoole** [comment]
This would seem to be locale related (aka the Accept-Language header), 
you can replicate this in browser by setting your browser preference to 
en first and then to de (German).

Am 16.11.2020 um 19:30 schrieb Mark Hallbauer:

**Comment by hallpower** [comment]
Hi @simonpoole, 

thanks! Yes, I was able to replicate the browser result by adding lang=en as a parameter in the Postman request. 

So then, this is more of a data issue between default locale and 'en'...

**Comment by simonpoole** [comment]
IMHO it is not so clear -why- it is different though. While ordering could well be expected to be different, why one result is missing completely is unclear and would warrant further investigation.

**Comment by lonvia** [comment]
We don't seem to be searching in the localised versions at all when default language is requested. That might be worth reconsidering.

**Comment by lonvia** [comment]
Fixed by #538.
